app/api/schema/feedbacks.py

```diff
@@ -35,3 +38,10 @@ class Meta:
                         schema='UserSchema',
                         type_='user'
                         )
+
+    @validates_schema
+    def validate_rating(self, data):
+        """Method to validate rating"""
+        rating = float(data['rating'])
+        if 0 <= rating <= 5:
+            raise ValidationError("Rating should be in range [0, 5].")
```

Comment:
There are already min max constraints in marshmallow, I recommend you going through marshmallow documentation